Dockerfile

```diff
@@ -1,7 +1,7 @@
-#---------------------------------------------------------------------------------------------
+# --------------------------------------------------------------------------------------------
```

Comment:
why this change

Reply:
Weird. Maybe something wrong with my IDE. Will revert this.